README.md

```diff
@@ -35,21 +35,22 @@ RIOT OS is an operating system for Internet of Things (IoT) devices. It is based
 Its features comprise
 * a preemptive, tickless scheduler with priorities
 * flexible memory management
-* high resolution timers
-* virtual, long-term timers
+* high resolution, long-term timers
+* support for AVR, MSP430, ARM7, and ARM Cortex-M on over 50 boards
```

Comment:
In general, I would try to keep the README as generic as possible. It's more like a teaser IMO.